luamocks/LuaSyncedCtrl.lua

```diff
@@ -19,202 +19,441 @@ stringMock ="TestString"
 tableMock ={exampletable= true}
 booleanMock =true
 functionMock =function (bar) return bar; end
+
+--==================================================================================================
+-- GameRulesParameter
 --==================================================================================================
+--[[ 
+---@param paramName string
+---@param losAccess? losAccess
+---@return string
+function Spring.SetGameRulesParam   (  paramName, losAccess )
```

Comment:
incorrect, there's the value before the name. unsure if mentioning the access makes any sense either since it is ignored for Game rules params

Reply:
While this file is being marked for deletion now, this is the current status:
`---@see Spring.GetGameRulesParams
---@param paramName string 
---@param paramValue number | string #numeric paramValues in quotes will be converted to number.
---@param losAccess? losAccess # not typically used in GameRules, see GetGameRulesParams, it will be ignored.
---@return nil
function Spring.SetGameRulesParam (  paramName, paramValue, losAccess )
	assert(type(paramName) == "string","Argument paramName is of invalid type - expected string");
	assert(losAccess == "private" | losAccess == "allied" | losAccess == "inlos" | losAccess == "inradar"
		| losAccess == "public" | losAccess == "typed" , "Argument losAccess is invalid");
return  nil
end`